src/test/resources/features/dcu/NoResponseNeededConfirmation.feature

```diff
@@ -3,23 +3,41 @@ Feature: No Response Needed Confirmation
 
   Background:
     Given I am logged into "DECS" as user "DCU_USER"
-    And I create a "MIN" case and move it to the "No Response Needed Confirmation" stage
-    And I load and claim the current case
 
   @DCUWorkflow @DCURegression
-  Scenario: User confirms the case does not require a response
-    When I click the NRN yes radio button
+  Scenario Outline: User confirms the case does not require a response
+    When I create a "<caseType>" case and move it to the "No Response Needed Confirmation" stage
+    And I load and claim the current case
+    And I click the NRN yes radio button
     And I click the "Finish" button
     Then the case should be closed
+    Examples:
+    | caseType  |
+    | MIN       |
+    | TRO       |
+    | DTEN      |
 
   @DCUWorkflow @DCURegression
-  Scenario: User selects that they dont agree that the case requires no response
-    When I click the NRN no radio button
+  Scenario Outline: User selects that they dont agree that the case requires no response
+    When I create a "<caseType>" case and move it to the "No Response Needed Confirmation" stage
+    And I load and claim the current case
+    And I click the NRN no radio button
     And I click the "Finish" button
     Then the case should be moved to the "Markup" stage
+    Examples:
+      | caseType  |
+      | MIN       |
+      | TRO       |
+      | DTEN      |
 
```

Comment:
I dont think we need to parameterise the validation scenarios, as those share the same page configuration for all workflows.